docs/how-to-guides/enqueueing-assets-in-the-editor.md

```diff
@@ -0,0 +1,117 @@
+
+For more information about why the Editor is iframed, please revisit the post [Blocks in an iframed (template) editor](https://make.wordpress.org/core/2021/06/29/blocks-in-an-iframed-template-editor/).
+
+## The Editor versus Editor content
+Before enqueueing assets in the Editor, you must first identify what you are trying to target. 
+
+Do you want to add styling or JavaScript to the user-generated content (blocks) in the Editor? Or do you want to modify the Editor user interface (UI) components or interact with Editor APIs? This could include everything from creating custom block controls to registering block variations. 
+
+There are different hooks to use depending on the answers to these questions, and if you are building a block or a theme, there are additional approaches to consider. Refer to the designated sections below. 
+
+## Editor scripts and styles
+
+Whenever you need to enqueue assets for the Editor itself (i.e. not the user-generated content), you should use the [`enqueue_block_editor_assets`](https://developer.wordpress.org/reference/hooks/enqueue_block_editor_assets/) hook coupled with the standard [`wp_enqueue_script`](https://developer.wordpress.org/reference/functions/wp_enqueue_script/) and [`wp_enqueue_style`](https://developer.wordpress.org/reference/functions/wp_enqueue_style/) functions. 
+
+Examples might be adding custom inspector or toolbar controls, registering block styles and variations in Javascript, registering Editor plugins, etc. 
+
+```php
+/**
+ * Enqueue Editor assets.
+ */
+function example_enqueue_editor_assets() {
+    wp_enqueue_script( 
+        'example-editor-scripts', 
+        plugins_url( 'editor-scripts.css', __FILE__ ) );
+    }
+    wp_enqueue_style( 
+        'example-editor-styles', 
+        plugins_url( 'editor-styles.css', __FILE__ ) );
+    }
+}
+add_action( 'enqueue_block_editor_assets, 'example_enqueue_editor_assets' );
+```
+
+While not the recommended approach, it's important to note that `enqueue_block_editor_assets` can be used to style Editor content for backward compatibility. See below for more details.
+
+## Editor content scripts and styles
+
+As of WordPress 6.3, all assets added through the [`enqueue_block_assets`](https://developer.wordpress.org/reference/hooks/enqueue_block_assets/) PHP action will also be enqueued in the iframed Editor. See [#48286](https://github.com/WordPress/gutenberg/pull/48286) for more details. 
+
+This is the primary method you should use to enqueue assets for user-generated content (blocks), and this hook fires both in the Editor and on the front end of your site. It should not be used to add assets intended for the Editor UI or to interact with Editor APIs. 
```

Comment:
Technically for backwards compatibility it fires twice in the editor. Once in the iframe and once outside the iframe. See https://github.com/WordPress/gutenberg/issues/53590

Reply:
Added note, thanks!